src/bluesearch/entrypoint/database/init.py

```diff
@@ -3,6 +3,8 @@
 
 import sqlalchemy
 
+from bluesearch.entrypoint.database.schemas import schema_articles, schema_sentences
+
```

Comment:
Same comment about import as in `add.py`.

Reply:
Done in 5787623.